src/app/lib/analyticsUtils/index.js

```diff
@@ -35,6 +51,49 @@ export const getDestination = statsDestination => {
     SPORT_PS_TEST: 598311,
   };
 
+  if (platform === 'amp') {
+    switch (statsDestination) {
+      case 'NEWS_PS': {
+        return getAmpDestination(
+          destinationIDs.NEWS_PS,
+          destinationIDs.NEWS_GNL,
+        );
+      }
+      case 'NEWS_PS_TEST': {
+        return getAmpDestination(
+          destinationIDs.NEWS_PS_TEST,
+          destinationIDs.NEWS_GNL_TEST,
+        );
+      }
+      case 'SPORT_PS_TEST': {
+        return getAmpDestination(
+          destinationIDs.SPORT_PS_TEST,
+          destinationIDs.SPORT_GNL_TEST,
+        );
+      }
+      case 'SPORT_PS': {
+        return getAmpDestination(
+          destinationIDs.SPORT_PS,
+          destinationIDs.SPORT_GNL,
+        );
+      }
+      case 'NEWS_LANGUAGES_PS': {
+        return getAmpDestination(
+          destinationIDs.NEWS_LANGUAGES_PS,
+          destinationIDs.NEWS_LANGUAGES_GNL,
+        );
+      }
+      case 'NEWS_LANGUAGES_PS_TEST': {
+        return getAmpDestination(
+          destinationIDs.NEWS_LANGUAGES_PS_TEST,
+          destinationIDs.NEWS_LANGUAGES_GNL_TEST,
+        );
```

Comment:
I'd suggest a bit more extensible structure compared to something like this, in it's current form it violates the 'open/closed' principle.

Maybe:
```
const geoVariants = {
  NEWS_PS : {
    PS: destinationIDs.NEWS_PS,
    GNL: destinationIDs.NEWS_GNL
  }
  ...
};

if(geoVariants[statsDestination]){
  return getAmpDestination(geoVariants[statsDestination]);
} else {
  return destinationIDs[statsDestination] || destinationIDs.NEWS_PS;
}
```
This allows new destinations to be added to the config array but the central algorithm itself is 'closed to modification' following the principle.